catalog/transformer/catalog-transformer-csv/src/main/java/ddf/catalog/transformer/csv/MetacardIterator.java

```diff
@@ -65,7 +67,19 @@ public Serializable next() {
     index++;
 
     if (attribute != null) {
-      return attribute.getValue();
+      if (attributeDescriptor.isMultiValued()) {
+        StringBuilder valuesBuilder = new StringBuilder();
+        List<Serializable> values = attribute.getValues();
+        for (int i = 0; i < values.size(); i++) {
+          if (i > 0) {
+            valuesBuilder.append(MULTIVALUE_DELIMITER);
+          }
+          valuesBuilder.append(values.get(i));
+        }
+        return valuesBuilder.toString();
```

Comment:
You can replace this block with `StringUtils.join(attribute.getValues(), MULTIVALUE_DELIMITER)`.

Reply:
Yeah I tried doing that initially but I ran into `Cannot resolve method join(String, List<Serializable>)`. Any idea how to easily coerce the `List<Serializable>` into a `List<? extends CharSequence>`? I could create a new list via a stream then use the `join()` method but it seemed easier to just duplicate the join logic instead.